src/Components/SuperAdmins/super-admins.module.css

```diff
@@ -1,89 +1,124 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
 .container {
-  flex: 1;
   display: flex;
   justify-content: center;
   align-items: center;
-  background-image: url('../../../public/assets/images/super-admin-background.png');
-  background-size: cover;
-  font-size: 1.5vw;
+  background-image: url('../../../public/assets/images/subscriptions-background.jpg');
 }
 
-.supAdmInput, .supAdmBtn {
-  font-size: 1.3vw;
-}
-
-.box {
-  color: white;
-  width: 85vw;
+.list {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  margin: 50px;
   background-color: rgba(55, 56, 103, 0.5);
-  padding: 2vw;
-  border-radius: 1vw;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  font-family: 'Roboto', sans-serif;
 }
 
-.table{
+.header {
   display: flex;
-  width: 95%;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.title {
+  color: white;
+  font-size: 40px;
+  line-height: 48px;
 }
 
-.supAdmTr:nth-child(even) {
-  background-color: rgba(55, 56, 103, 0.7);
+.btn {
```

Comment:
You shouldn't have styles for buttons, their deppend on the shared component

Reply:
They were left over from the original css and I forgot to remove them. They are currently not used.